portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,33 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+// Fetches comments from the server and adds them to the DOM.
+function LoadComments(){
+  var max=getMax();
+  fetch('/data?user-choice='+max).then(response => response.json()).then((comments) => {
+  const commentListElement=document.getElementById('comment-list');
+  commentListElement.innerHTML=' ';
+  comments.forEach((comment)=>{
+  commentListElement.appendChild(createListElement(comment));
+    });
+  });
+}
+
+// Creates an element that represents a comment
+function createListElement(comment){
+  const commentListElement=document.createElement('li');
+  commentListElement.innerText=comment.comment
+  return commentListElement;
+}
+
+function getMax(){
+  var temp=document.getElementById("maxwant");
```

Comment:
nit: I think a name like `maxElement` might be better than `temp`.  Also there should be spaces before and after the `=`

Reply:
Addressed in latest commit.